libs/wire-subsystems/src/Wire/UserGroupSubsystem/Interpreter.hs

```diff
@@ -279,7 +279,7 @@ deleteGroup deleter groupId =
       unless (isAdminOrOwner (member ^. permissions)) $
         throw UserGroupNotATeamAdmin
 
-      Store.deleteUserGroup team groupId >>= noteS @'UserGroupNotFound
+      Store.deleteUserGroup team groupId >>= note UserGroupNotFound
```

Comment:
shouldn't the note be removed entirely?  in what case do i want to still get a not found?

Reply:
Changed!